tests/plugins/clipboard/manual/pasteimagedisable.md

```diff
@@ -0,0 +1,17 @@
+@bender-ui: collapsed
+@bender-tags: 4.17.0, clipboard, 4874
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, basicstyles, image, clipboard, sourcearea
+
+ 1. Paste an image into the editor.
+
+    Observe that the image is inserted and that no "Custom image paste handling" entry is written to "log output" on the page.
```

Comment:
I'd split it into two tests: the one that will test the plugin with handling image pasting and one without it. It should be more readable than putting both of these checks inside one test. Could you split the test?

Reply:
I added unit tests to cover scenarios where image paste handling has been disabled using the new `clipboard_handleImagePasting` flag - see tests/plugins/clipboard/pasteimagedisabled.
Native image paste handling (when enabled) is already covered by tests/plugins/clipboard/pasteimage.

Do you still want manual test coverage, or should I just remove it again ?

Reply:
Oh, worth mentioning; I took the liberty to move some code from tests/plugins/clipboard/pasteimage.js into tests/plugins/clipboard/_helpers/pasting.js to reduce duplicate code. I needed the exact same code for the new unit test in tests/plugins/clipboard/pasteimagedisabled.js

Reply:
> Do you still want manual test coverage, or should I just remove it again ?

It's always better to have an additional manual test. For me manual tests are for people and unit ones – for machines.

Reply:
Alright, I'll refacter the manual test as you suggested. Thanks 😊

Reply:
There. The new manual test (tests/plugins/clipboard/manual/pasteimagedisable.md) is now only about testing the new `clipboard_handleImagePasting` option, as support for image pasting is already covered with tests/plugins/clipboard/manual/pasteimage.md.